sdcm/utils/version_utils.py

```diff
@@ -552,3 +553,16 @@ def inner(*args, **kwargs):
             raise MethodVersionNotFound("Method '{}' with version '{}' is not supported in '{}'!".format(
                 func.__name__, scylla_version, cls_self.__class__.__name__))
         return inner
+
+
+def get_relocatable_pkg_url(scylla_version: str) -> str:
+    relocatable_pkg = ""
+    if "build-id" in scylla_version:
+        try:
+            scylla_build_id = scylla_version.split('build-id')[-1].split()[0]
```

Comment:
nitpick: add an example of the `scylla_version` format, to be clear the `split`s in here